firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -4079,7 +4097,7 @@ extension String {
             public static let switchOffText = MZLocalizedString(
                 key: "Menu.EnhancedTrackingProtection.SwitchOff.Text.v128",
                 tableName: "EnhancedTrackingProtection",
-                value: "Protections are OFF. We suggest turning it back on.",
+                value: "Protections are OFF. We suggest turning them back on.",
```

Comment:
since we're modifying this string, should we create a new key? cc: @Delphine

Reply:
@petruSt24 should we add the old string to the old struct indicated in the wiki docs:
https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#steps-1